module-4/remove_metadata/README.md

```diff
@@ -68,7 +62,7 @@ This function removes metadata from a PDF file using ExifTool.
   - Executes the command using `subprocess.run`.
```

Comment:
add an small explanation what that means and why you want the program to run in a subprocess.